app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -63,14 +66,25 @@ const val DEFAULT_KEY_MODIFICATIONS = ""
 data class AppSettings(
     @PrimaryKey(autoGenerate = true) val id: Int,
     @ColumnInfo(
-        name = "key_size",
-        defaultValue = DEFAULT_KEY_SIZE.toString(),
+        name = "auto_size_keys",
+        defaultValue = DEFAULT_AUTO_SIZE_KEYS.toString(),
     )
-    val keySize: Int,
+    val autoSizeKeys: Int,
```

Comment:
New columns need to be at the end, in the correct order.

Reply:
Really? Because this seemed to work in some testing.

I mean sure, and I'm no expert in sqlite or whatever this db wrapper layer is, I'm just confused why it seemed to work.